plvp/types.go

```diff
@@ -53,13 +53,15 @@ type LocalConfig struct {
 	StartScamp   *bool   `flag:"start-scamper"`
 	Host         *string `flag:"host"`
 	RootCA       *string `flag:"root-ca"`
+	Interface    *string `flag:"interface"`
```

Comment:
On line 47 of main.go, the interface flag is specified as just `-i`. Should this be just `flag:"i"`?